SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -449,6 +449,15 @@ void ASpatialFunctionalTest::RegisterFlowController(ASpatialFunctionalTestFlowCo
 	FlowControllers.Add(FlowController);
 }
 
+void ASpatialFunctionalTest::DeregisterFlowController(ASpatialFunctionalTestFlowController* FlowController)
+{
+	if (FlowController->WorkerDefinition.Type == ESpatialFunctionalTestWorkerType::Client)
```

Comment:
maybe an error msg if it's not a client controller? Though I guess we may want to extend this to disconnecting servers as well at some point

Reply:
I've added an error, for now, we can always remove it alter if we want to extend the functionality.